src/uucore/src/lib/features/format/spec.rs

```diff
@@ -484,6 +490,21 @@ fn resolve_asterisk_maybe_negative<'a>(
     }
 }
 
+fn resolve_asterisk_precision<'a>(
```

Comment:
would be nice to add a comment explaining what this function is doing
and write unit tests (if not too complex)

Reply:
Sure thing, added doc and unit tests. Thanks for the review!